.gitignore

```diff
@@ -31,6 +31,9 @@ _ReSharper*/
 *.[Rr]e[Ss]harper
 *.DotSettings.user
 
+# CodeRush
```

Comment:
These changes to `.gitignore` belong in a separate PR.

Reply:
> would rename ToSourceKind ToBreakingSource to make it obvious that the source is a broken as a sequence

`ToSourceKind` _doesn't_ trip on `SourceKind.Sequence`. Maybe the enum values should be renamed?

Reply:
I've addressed your suggestion in cdbec3b904fc030ea5988fd101e3e69f7787a555.